labgrid/remote/client.py

```diff
@@ -820,13 +820,15 @@ def main():
     subparser = subparsers.add_parser('io',
                                       help="change (or get) a digital IO status")
     subparser.add_argument('action', choices=['high', 'low', 'get'])
+    subparser.add_argument('-v', '--verbose', action='count', default=0)
```

Comment:
Maybe it's useful to always enable the step reporting when using state support? Do we need to switch these two verbose mechanisms separately?